src/Payment.php

```diff
@@ -86,4 +86,11 @@ public function bankTransfer()
 
         return $this->request('GET', $relativeUrl);
     }
+
+    public function createRecurring($attributes = array())
+    {
+        $relativeUrl = $this->getEntityUrl() . 'create/recurring';
+ 
+        return $this->request('POST', $relativeUrl, $attributes);   
+    }
```

Comment:
I guess you already done this in cards module. If you have done this before will get Git Conflicts while merging.
So please check once.

Reply:
yes i have done this in card module but here i change the variable name